core/controller/src/main/scala/whisk/core/loadBalancer/ActivationThrottle.scala

```diff
@@ -120,22 +120,25 @@ class ActivationThrottle(config: WhiskConfig)(
         }
 
     Scheduler.scheduleWaitAtLeast(healthCheckInterval) { () =>
-        for {
-            concurrencyLimits <- getConcurrencyLimits
+        // Fall back to the current version of the limits
+        val concurrency = getConcurrencyLimits.recover {
+            case _ => userActivationLimits
+        }.map { concurrencyLimits =>
+            userActivationLimits = concurrencyLimits
+        }
+
+        val activations = for {
```

Comment:
If one of both calls fail, the calculation would be invalid anyway.

Your comment made me notice though, that above "fallback" is unnecessary. Removing.